config/api.yml

```diff
@@ -268,7 +268,7 @@
       :post:
       - :name: edit
         :identifier: host_edit
-        :disabled: true
+        :disabled: false
```

Comment:
Ah yes I meant to ask about that. No problem!

Rats, can't believe I forgot about multiple updates. I'll add a test and update